requirements-periodic-scraping.txt

```diff
@@ -0,0 +1,10 @@
+attrs==17.3.0
+beautifulsoup4==4.6.0
+boto3==1.7.4
+bs4==0.0.1
+lxml==4.1.1
+pdfminer.six==20170720
+psycopg2-binary==2.7.4
+pyahocorasick==1.1.6
+requests==2.18.4
+Scrapy==1.4.0
```

Comment:
Note: this doesn’t give you a deterministic set of pins, because you’re not pinning any of the transitive dependencies.

Better: use something like pip-tools or pipenv to specify the top-level packages you care about, and then create a lock file that pins everything.

Reply:
👍